README.md

```diff
@@ -91,6 +91,8 @@ Usage of moroz:
     	path to config folder (default "../../configs")
   -event-logfile string
     	path to file for saving uploaded events (default "/tmp/santa_events")
+  -persist-events
+      Enable writing events to disk (default `true`)
```

Comment:
The PR description says the default value is set to `false` but it's `true` here. I think leaving the default set to `true` makes sense and ensures we won't break others when they upgrade.

Reply:
Sorry about that, I forgot to change the PR description. I agree with you on setting the default to true. 

Do you feel that `-persist-events` is descriptive enough? I did struggle with finding a suitable name for the flag, this was the best I could come up with.